jinahub/indexers/searcher/HnswlibSearcher/tests/unit/test_searcher.py

```diff
@@ -80,6 +80,12 @@ def test_index():
     assert index._index.element_count == NUM_DOCS
     assert set(index._ids_to_inds.keys()) == set(da1.get_attributes('id'))
 
+    da3 = DocumentArray([Document() for emb in embeddings])
```

Comment:
Can you move this out to a separate test function? I prefer having each unit test as atomic as possible

Reply:
addressed in latest commits